modules/logging_config/log.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# TODO: idk what the channel names are
+CHANNELS=(can_primary can_secondary)
+declare -A PIDS
+CURR_MP=""
+
+timestamp() {
```

Comment:
Instead of manually doing timestamps you can use the -t option with candump, which I'm guessing is suportted by cantools (but you should double check).